incubator/virtualcluster/pkg/vn-agent/server/translate.go

```diff
@@ -74,11 +70,17 @@ func translateRawQuery(req *restful.Request, containerName string) {
 				query.Add("stdout", "false")
 			}
 		case "tailLines":
+			// for log option --tail
 			klog.V(5).Infof("the tail parameter is %s", v[0])
 			if v[0] != "all" {
 				// "all" is the same as omitting tail
-				query["tailLines"] = []string{v[0]}
+				query.Add("tailLines", v[0])
```

Comment:
Can we use query.Add(k, v[0]) as well?

Reply:
Yes. I just found out that "tailLines" can be handled in the same way as other options